### ISSUE TITLE
antiadblock

### ISSUE THREAD

**Issue by ghajini** [reek/anti-adblock-killer#2831]
@mapx- ,@uBlock-user
here on
https://www.flashx.tv/1ek77pp2m6f1.html
click on proceed to video
will get warning
![screenshot 128](https://cloud.githubusercontent.com/assets/20338483/22853093/91d56ef2-f072-11e6-9808-334d4dc8a665.png)

also too much popups
ublock origin 1.11.0

any ublock specific script??
note = clear cookies ,restart browser to produce problem

**Comment by uBlock-user** [comment]
https://github.com/reek/anti-adblock-killer/issues/2639#issuecomment-270577042

**Comment by ghajini** [comment]
still get warning
used both
||flashx.tv/js/code.js

`flashx.tv##script:inject(abort-on-property-read.js, jQuery.adblock)`

![screenshot 130](https://cloud.githubusercontent.com/assets/20338483/22853202/d3c3770c-f075-11e6-98ef-2ad147321e2e.png)

**Comment by uBlock-user** [comment]
Works fine for me. Clear your cache, site storage data and cookies, then try again.

**Comment by ghajini** [comment]
nothing works 
tried w/ ,  w/o vpn on fresh chrome profile with clearing cookies
![screenshot 131](https://cloud.githubusercontent.com/assets/20338483/22853249/63b7b944-f077-11e6-8ee4-c804bb557180.png)

**Comment by uBlock-user** [comment]
I honestly can't say.

Loads fine on my side - https://i.gyazo.com/d8b0fca6a06dea8a1d1283c4d91d7cb0.png

**Comment by ghajini** [comment]
no, iam circumventing my isp if they are blocking any scripts on page
it happens with q.gs issue which in uassets page..there gorhill pointed my isp interference 
also on ur end you are using umatrix

**Comment by uBlock-user** [comment]
I run 3 extensions only uBO, uMX and uBO-Extra, uMX only acts as a firewall and it won't make any difference when it comes to this.

**Comment by ghajini** [comment]
why youtube not work with umatrix

**Comment by uBlock-user** [comment]
Read - https://github.com/gorhill/uMatrix#warnings
